mentors/python/s.json

```diff
@@ -141,5 +134,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I enjoy using Python for work and fun.  There's no other language that allows me to go from an idea to working code as fast."
+  },
+  {
+    "github_username": "pyropy",
```

Comment:
This is the wrong JSON file for you to be inserting your bio.  Since your GitHub name begins with `p` you should be updating the file `p.json` instead.

Reply:
Sure thing. Sorry I've missread instructions. I'll remove it and create new PR for my bio, leaving the formatted json only.